### PR TITLE
added audio

### PR DESCRIPTION
**Pull request by bem-t**
added audio

### CODE REVIEW

**Comment by adilet-nasirov** [comment]
That's great Bermet!

File: main.js [hunk 1]
```diff
@@ -1,63 +1,66 @@
...
-        searchResult.innerHTML += `
+  console.log(data);
+  if (data.total === 0) {
+    showNotification("Try another one or you can write it )))");
+  } else {
+    searchSection.classList.add("search-top");
+    searchResult.style.display = "block";
+  }
+  data.data.forEach((lyrics) => {
+    console.log(lyrics.title);
```
Comment (tagaism) [inline main.js]: please, delete all debugging tracks (console.logs).

File: main.js [hunk 2]
```diff
@@ -1,63 +1,66 @@
...
-    if(data.total === 0) {
-        showNotification('Try another one or you can write it )))');
-    } else {
-        searchSection.classList.add('search-top');
-        searchResult.style.display = 'block';
-    }
-    data.data.forEach(lyrics => {
-        console.log(lyrics.title)
-        searchResult.innerHTML += `
+  console.log(data);
```
Comment (tagaism) [inline main.js]: this one should be deleted as well.

**Review by tagaism — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (bem-t) [reply main.js]: Ok. Thank you. All debugging tracks are deleted.

Comment (tagaism) [reply main.js]: It is still shown.
Or you haven't pushed yet?

**Pull request merged**